client/src/components/navbar/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component }from 'react';
+import ReactDOM from 'react-dom';
+
+import './style.css'    
+
+class Navbar extends Component{
+    constructor() {
+         super();
+         this.state={
+         activeTab: 'flights'
+        }
+
+        this.handleTabChange = this.handleTabChange.bind(this);
+    }
+
+    handleTabChange(activeTab) {
+        this.setState({
+         ...this.state,
+        activeTab
+        })
+    } 
+
+
+    render(){
+        return(
+      <div>    
+         <div className = "nav">
+         <div className ="user">
```

Comment:
I think these two divs need closing tags or need to be self-terminating (i.e. `<div className='user' />`)